### ISSUE TITLE
not connecting base?

### ISSUE THREAD

**Issue by EPiC-APOC** [ScriptTiger/Unified-Hosts-AutoUpdate#33]
sometimes the script is stuck in somekind of loop, like this:
```Checking for script updates...
Script updates currently disabled
The Unified Hosts is already installed in your local hosts file
Checking for Unified Hosts updates...
Download Retry 1: base...
Download Retry 2: base...
Download Retry 3: base...
Download Retry 4: base...
Download Retry 5: base...
Download Retry 6: base...
Download Retry 7: base...
Download Retry 8: base...
Download Retry 9: base...
Download Retry 10: base...
Download Retry 11: base...
Download Retry 12: base...
Download Retry 13: base...
Download Retry 14: base...
Download Retry 15: base...
Download Retry 16: base...
Download Retry 17: base...
Download Retry 18: base...
Download Retry 19: base...
Download Retry 20: base...
Download Retry 21: base...
Download Retry 22: base...
Download Retry 23: base...
Download Retry 24: base...
Download Retry 25: base...
Download Retry 26: base...
Download Retry 27: base...
Download Retry 28: base...
Download Retry 29: base...
Download Retry 30: base...
Download Retry 31: base...
Download Retry 32: base...
Download Retry 33: base...
Download Retry 34: base...
Download Retry 35: base...
Download Retry 36: base...
Download Retry 37: base...
Download Retry 38: base...
Download Retry 39: base...
Download Retry 40: base...
Download Retry 41: base...
Download Retry 42: base...
Download Retry 43: base...
Download Retry 44: base...
Download Retry 45: base...
Download Retry 46: base...
Download Retry 47: base...
Download Retry 48: base...
Download Retry 49: base...
Download Retry 50: base...
Download Retry 51: base...
Download Retry 52: base...
Download Retry 53: base...
Download Retry 54: base...
Download Retry 55: base...
Download Retry 56: base...
Download Retry 57: base...
Download Retry 58: base...
Download Retry 59: base...
Download Retry 60: base...
Download Retry 61: base...
Download Retry 62: base...
Download Retry 63: base...
Download Retry 64: base...
Download Retry 65: base...```

**Comment by ScriptTiger** [comment]
My script only checked connectivity once while checking script version. However, you have script updates disabled, so it was skipping a connectivity check altogether and getting stuck in a loop when there was no connectivity. This has been fixed in release 1.27.

When you have time, please verify this resolves your issue.

**Comment by EPiC-APOC** [comment]
Correct, I have this disabled, I'll test your v1.27

Thanks
